contracts/src/SafeExitTimelock.sol

```diff
@@ -5,7 +5,7 @@ import "@openzeppelin/contracts/governance/TimelockController.sol";
 
 /// @title TimelockController
 /// @notice A timelock controller for the StakeTable contract
-contract Timelock is TimelockController {
+contract SafeExitTimelock is TimelockController {
```

Comment:
It seems a bit strange to have two contracts that are exactly the same just with two different names and create bindings for both. Is there a downside to re-using the contract? If we want different rust types we could create newtype wrappers around the Timelock bindings. @alxiong wdyt?

Reply:
i agree that using one would be better from a code maintenance pov. I was optimizing for explorers and multisig UIs so that the viewer can easily distinguish between contracts via the deployed name (i.e. seeing `SafeExitTimelock` on the explorer and `OpsTimelock` for their respective contract addresses). Also this allows for any implementation divergence in the future.

Reply:
Ok. You mean when someone looks at the verified source code on etherscan, for example? Or where would this name show up? It's not part of the contract binary.